js/binance.js

```diff
@@ -6043,6 +6045,58 @@ module.exports = class binance extends Exchange {
         };
     }
 
+    async repayMarginLoan (code, amount, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+        }
+        const currency = this.currency (code);
+        const request = {
+            'asset': currency['id'],
+            'amount': amount,
+        };
+        const defaultMargin = this.safeString (params, 'marginMode', 'cross'); // cross or isolated
```

Comment:
Let's not forget to omit `marginMode` from `params`:

```suggestion
        const defaultMargin = this.safeString (params, 'marginMode', 'cross'); // cross or isolated
        params = this.omit (params, 'marginMode');
```